app.rb

```diff
@@ -0,0 +1,104 @@
+require_relative 'book'
+require_relative 'person'
+require_relative 'student'
+require_relative 'rental'
+require_relative 'teacher'
+
+class App
+  attr_accessor :books, :persons, :rentals
+
+  def initialize
+    @books = []
+    @persons = []
+    @rentals = []
+  end
+
+  # List all books.
+  def list_books
+    puts 'List of Books:'
+    @books.each do |book|
+      puts "📚 Title: #{book.title} by #{book.author}"
+    end
+  end
```

Comment:
| @mohashyne List of Books | Microverse List of Books |
|-|-|
| ![image](https://github.com/mohashyne/school_library/assets/93039359/2ee60d8f-59a1-4cbf-9699-df7fdcd9720c) | ![image](https://github.com/mohashyne/school_library/assets/93039359/8d986bf7-96a5-42f1-bdd9-fca6cd8bd28f) |

- [ ] It would be nice if you make your console app behave the same way as in the given design by microverse 👍 because I can see when we render the books it shows us an object and in the same line we see no author 😨 which is alarming because it can create confusion for the user and is also not following the given design 🤔 Kindly make your console app return the books in the same format which is shown in the microverse console app 👍

Reply:
- Well resolved 👍🏽